doc/manuals/user/filtering.md

```diff
@@ -188,7 +188,26 @@ described in detail in [the following section](geolocation.md#geo-located-querie
 
 [Top](#top)
 
-### String filter
+### String filters
+
+Two different types of string filters are supported:
+* filtering over attributes (scope type: FIWARE::StringQuery) and
+* filtering over the metadatas of attributes (scope type: FIWARE::StringQuery::Metadata).
+
+These scopes allow to express filtering conditions such as equality, unequality,
+greater/less than, in-array, range or existence (of attributes and metadata of attributes).
+
+There isn't any URL parameter corresponding to these filters in NGSIv1.
+In NGSIv2 they correspond to the `q` and `mq` URI parameters.
+
+For a detailed syntax description of the `value` or `q` and `mq` parameters, see [NGSIv2 specification
```

Comment:
"value" should be in normal text (or I'm missing the reason to use code-style in this case...)